themes/wporg-translate-events-2024/blocks/components/event/title.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Wporg\TranslationEvents\Theme_2024;
+use Wporg\TranslationEvents\Translation_Events;
+use Wporg\TranslationEvents\Urls;
+
+register_block_type(
+	'wporg-translate-events-2024/components-event-title',
+	array(
+		// phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.Found
+		'render_callback' => function ( array $attributes ) {
+			$event = Translation_Events::get_event_repository()->get_event( $attributes['id'] );
+			$event_ids = $attributes['event_ids'] ?? array();
+			$user_id = get_current_user_id();
+			$current_user_attendee_per_event = Translation_Events::get_attendee_repository()->get_attendees_for_events_for_user( $event_ids, $user_id );
+			$current_user_attendee = $current_user_attendee_per_event[ $event->id() ] ?? null;
+
+			if ( $current_user_attendee ) {
+				$event_flag = 'Attending';
+				if ( $current_user_attendee->is_host() ) {
+					$event_flag = 'Host';
```

Comment:
I think this block should only render the title, and we should have another block that renders the attendance/hosting labels. We might not want to always render the attendance/hosting labels every time we render the title.